src/app.rs

```diff
@@ -96,7 +98,7 @@ mod imp {
                 if let Some(settings) = gtk::Settings::default() {
                     settings.set_gtk_font_name(Some("Segoe UI 10"));
                 }
-            } else if cfg!(target_os = "macos") {
+            } else if cfg!(target_os = MACOS) {
```

Comment:
This is not going to compile either for similar reasons.

`cfg!` is a macro that is evaluated during compilation in order to choose whether to generate the binary code for the if branch or not. Therefore, it doesn't have access to the `MACOS` variable, because the variable only exists after the program is running, but the if has to be evaluated here during compilation, so the variable doesn't exist yet.

I get the intentions of trying to use common strings, but in this case, it doesn't work as it looks from the outside. Don't consider this string as code, but as data (imagine it like a `type` parameter in a JSON object).